packages/drawer/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@littlespoon/drawer",
+  "version": "1.0.0",
+  "description": "Little Spoon drawer",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "tsc",
+    "build:flowtypes": "flowgen lib && rm -rf lib && mv exports lib && yarn build",
+    "clean": "rm -rf coverage exports lib",
+    "lint": "eslint .",
+    "lint:fix": "yarn lint --fix",
+    "lint:tsc": "tsc --noEmit",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@littlespoon/button": "1.7.2",
+    "@littlespoon/icons": "1.9.0",
```

Comment:
Adding a todo comment here since after #607 is merged, this will need to be updated

Reply:
Updated in 6a4835b2b345ecbff752c35a51576ef134a9f4ee